fs/xfstests.py.data/local.config

```diff
@@ -0,0 +1,4 @@
+export TEST_DEV=/dev/loop0
+export TEST_DIR=/mnt/test
+export SCRATCH_DEV=/dev/loop1
+export SCRATCH_MNT=/mnt/scratch
```

Comment:
Does the user need to edit this file frequently ?
If yes, can it be handled with yaml ?